scene/gui/rich_text_label.h

```diff
@@ -340,10 +340,17 @@ class RichTextLabel : public Control {
 	void set_bbcode(const String &p_bbcode);
 	String get_bbcode() const;
 
+	void set_text(const String &p_string);
+
 	void set_visible_characters(int p_visible);
 	int get_visible_characters() const;
 	int get_total_character_count() const;
 
+	void set_percent_visible(float p_percent);
+	float get_percent_visible() const;
+
+	float percent_visible;
```

Comment:
This should go further up next to `int visible_characters;` as a private member.